docs/release-notes/2.41.1.md

```diff
@@ -33,5 +33,6 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
 
 
 ## Bugs Fixed
+- [PMM-12425](https://jira.percona.com/browse/PMM-12425) - [QAN] Resolved issue with Query Analytics when using ‘now’ in the form instead of a specific time for the time range.
```

Comment:
```suggestion
- [PMM-12425](https://jira.percona.com/browse/PMM-12425) - [QAN]: The **Absolute time range** time picker on the **Query Analytics (QAN)** page no longer displays an error when using 'now' instead of a specific timestamp for filtering dashboard data.


```